src/invoice/invoice-table.component.tsx

```diff
@@ -96,20 +97,32 @@ const InvoiceTable: React.FC<InvoiceTableProps> = ({ bill, isLoadingBill }) => {
           total: convertToCurrency(item.price * item.quantity, defaultCurrency),
           actionButton: (
             <span>
-              <Button
-                data-testid={`edit-button-${item.uuid}`}
-                renderIcon={Edit}
-                hasIconOnly
-                kind="ghost"
-                iconDescription={t('editThisBillItem', 'Edit this bill item')}
-                tooltipPosition="left"
-                onClick={() => handleSelectBillItem(item)}
-              />
+              {canEditBill ? (
+                <Button
+                  data-testid={`edit-button-${item.uuid}`}
+                  renderIcon={Edit}
+                  hasIconOnly
+                  kind="ghost"
+                  iconDescription={t('editThisBillItem', 'Edit this bill item')}
+                  tooltipPosition="left"
+                  onClick={() => handleSelectBillItem(item)}
+                />
+              ) : (
+                <Button
+                  data-tested={`edit-button-${item.uuid}`}
+                  renderIcon={Edit}
+                  hasIconOnly
+                  kind="ghost"
+                  disabled
+                  iconDescription={t('cannotEditThisBill', 'You can not edit this bill')}
+                  tooltipPosition="left"
+                />
+              )}
```

Comment:
We don’t need to conditionally render a ghost button. Just disabling the button is sufficient, like so:
```suggestion
            <Button
              data-testid={`edit-button-${item.uuid}`}
              renderIcon={Edit}
              hasIconOnly
              kind=“ghost”
              iconDescription={t('editThisBillItem', 'Edit this bill item’)}
              tooltipPosition=“left”
              onClick={() => handleSelectBillItem(item)}
              disabled={bill?.status !== ‘PENDING’}
            />
```